### DISCUSSION TITLE
Showcase: Collection of WP RIG based websites here (post yours!)

### DISCUSSION THREAD

**Discussion by LukaszJaro** [wprig/wprig#692] [Show and tell]
Also list if v1 or v2!

**Comment by LukaszJaro** [comment]
Michelle theme using wprig but modified version, was featured in a recent wptavern post.

https://wptavern.com/the-michelle-wordpress-theme-launches-with-dozens-of-block-patterns-and-styles

> I opened a code editor and loaded my custom starter theme, which is a modern code mashup inspired by WP Rig, Mythic, and my own code perfected over years, and it successfully powers all of my new themes released since last year.

-https://www.webmandesign.eu/portfolio/michelle-wordpress-theme/

**Comment by digisavvymachine** [comment]
We build the site for Huntington Hospital: https://huntingtonhospital.org, although we are no longer maintaining it.

**Comment by jfmacdonald** [comment]
We built and maintain the site [ypinfrastructure.com]( https://ypinfrastructure.com/) for Young People in Infrastructure with a custom theme based on WP RIG 2.0. Glad to see a new maintenance team for WP RIG.
